dist/infragram.js

```diff
@@ -1,14 +1,15 @@
-function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+ isVideo = false,isCamera=false; // Turns off camera feed
```

Comment:
Are these global variables? Could we set them actually inside of the `camera.js` file instead, and use `var`?

https://github.com/publiclab/infragram/blob/3f5f87aefef84d92f87eed3e4b22a0cd6edc8bf0/src/io/camera.js#L3